xpmir/papers/monobert/lr_6e-7.yaml

```diff
@@ -0,0 +1,13 @@
+type: monobert_lr_6e-7
```

Comment:
no need to have one file per setting / better to allow overriding the default in the experiment file